styles.css

```diff
@@ -0,0 +1,75 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  width: 375px;
+  height: 718px;
+  padding: 20px;
+  background: #fff;
+}
```

Comment:
- [ Optional ] Please consider fixing the horizontal scrolling at width `375px` on mobile devices. I think it's caused because of the fixed width of `375px` in the `.containter` styling. Please remove the styling `width: 375px` to fix this issue please see the screenshot below for more details.
### Menu Icon missing because of horizontal scroll
![Screenshot from 2022-12-01 13-23-02](https://user-images.githubusercontent.com/16392046/205029610-a33a9b79-a52b-409b-96fa-6f804dd6dbff.png)